base/src/sgpp/base/grid/generation/StandardGridGenerator.cpp

```diff
@@ -35,7 +36,7 @@ void StandardGridGenerator::regular(size_t level, double T) {
 }
 
 void StandardGridGenerator::regularInter(size_t level,
-                                         const std::vector<std::vector<size_t>>& terms, double T) {
+                                         const std::set<std::set<size_t>>& terms, double T) {
```

Comment:
This needs a corresponding `%template` in the corresponding `base.i`s, otherwise this would probably (?) break compatibility with Python etc.